File: java/Overloading.java
public class Overloading {
    public static void main(String[] args) {
        // beda konstruktor, tetap jadi objek manusia
        Manusia ujang = new Manusia(20, "Ujang");
        Manusia udin = new Manusia("Udin", 21);
        System.out.println(ujang);
        System.out.println(udin);

        // set umur, tapi beda jenis parameter
        System.out.println("---");
        ujang.setUmur("30");
        udin.setUmur(25);
        System.out.println(ujang);
        System.out.println(udin);
    }
}

File: python/Overloading.py
from Manusia import Manusia

# beda konstruktor, tetap jadi objek manusia
# sebetulnya python tidak support multiple constructor, tapi kita bisa pilih parameter mana yang mau dimasukkan
ujang = Manusia("Ujang", 20)
udin = Manusia(umur = 21, nama = "Udin");
print(ujang.toString())
print(udin.toString())

# set umur, tapi beda jenis parameter
print("---");
ujang.setUmur("30")
udin.setUmur(25)
print(ujang.toString())
print(udin.toString())